src/Apps/Collect/CollectionApp.tsx

```diff
@@ -122,6 +131,9 @@ export const CollectionAppFragmentContainer = createFragmentContainer(
         relatedCollections {
           ...RelatedCollectionsRail_collections
         }
+        linkedCollections {
```

Comment:
What's the difference between related collections and linked collections?

Reply:
[Related collections](https://github.com/artsy/kaws/blob/c45a8899867a15af35a5c4888e6d772c7e6c683b/src/Resolvers/Collections.ts#L93-L124) is based on either the `artist_ids` field in a collection query or the category. This is used to populate the related collection rail that's at the bottom of collection pages. Linked collections are hand curated collections developed by the marketing team. The presence of linked collection is essentially what makes a collection a hub. Not every collection will have linked collection but every collection will have related collections.

Reply:
Ohh, that makes sense. I think `curatedCollections` might be a better name in that case, just so its a little more clear how it's different from the `relatedCollections`. But I understand that renaming things while meeting Relay's naming conventions can be precarious.